### ISSUE TITLE
Some editing cleanup following cred man merge

### ISSUE THREAD

**Issue by kpaulh** [w3c/webauthn#414]
Editorial cleanup for things like:
- extra backslash 4.1.3: \\[[Create]]
- [[discovery]] incorrectly in code font
- Inconsistently describing the two methods in 4.1.3 and 4.1.4...
i.e.,
"Create a new credential - ScopedCredential’s \\\[[Create]](options) method"
vs.
"ScopedCredential::\[[DiscoverFromExternalSource]](options)"

**Comment by kpaulh** [comment]
Inconsistently describing the two methods in 4.1.3 and 4.1.4...
i.e.,
Create a new credential - ScopedCredential’s \[[Create]](options) method
vs.
ScopedCredential::[[DiscoverFromExternalSource]](options)

**Comment by equalsJeffH** [comment]
Technical items:
* MakeCredentialOptions.excludeList should be declared as:
`sequence<ScopedCredentialDescriptor> excludeList = [];`  // it has lost the default empty seq "[]"

**Comment by equalsJeffH** [comment]
also, since PR #423 was merged, there's now these bikeshed errors..
```
> curl https://api.csswg.org/bikeshed/ -F file=@index.bs -F force=1 -F output=err
\033[1;33mLINK ERROR:\033[0m No 'idl' refs found for 'displayName' that are marked for export.
&lt;a data-link-type="idl" data-lt="displayName" data-link-for="ScopedCredentialEntity">displayName&lt;/a>
\033[1;33mLINK ERROR:\033[0m No 'dfn' refs found for 'user verified'.
&lt;a data-link-type="dfn" data-lt="User Verified">User Verified&lt;/a>
\033[7;32m ✔ \033[0m Successfully generated, with 2 linking errors
```

**Comment by equalsJeffH** [comment]
@kpaulh noted above:
> extra backslash 4.1.3: \[[Create]]

there's extra backslashes in various places, all of which are apparently implicit refs to things in [credman](https://github.com/w3c/webauthn/issues/414), and the "\"s are there to impede bikeshed from attempting to cross-reference them -- which produces errors because those credman things are not yet entered into ([IIUC](https://en.wiktionary.org/wiki/IIUC)) http://www.specref.org/.

**Comment by equalsJeffH** [comment]
@kpaulh noted above:
> [[discovery]] incorrectly in code font

hm, what you mean is that it is not in bold italic font as other \<dfn>s are?  IIUC that is on purpose because the \<dfn> of the [[discovery]] internal slot <https://w3c.github.io/webappsec-credential-management/#dom-credential-discovery-slot> is actually over in credman... ?

**Comment by equalsJeffH** [comment]
@kpaulh noted above:
>Inconsistently describing the two methods in 4.1.3 and 4.1.4...
i.e.,
"Create a new credential - ScopedCredential’s \\\[[Create]](options) method"
vs.
"ScopedCredential::\[[DiscoverFromExternalSource]](options)"

hm, yes, IIUC, we ought to pick one of the above approaches and apply it to the other.  It seems the "::" notation is not formally part of webidl at this time, see https://github.com/heycam/webidl/issues/354, so perhaps we ought to go with the other approach (which is ugly...) ?  Or maybe the "::" notation will soon return to webidl?

/cc @mikewest

**Comment by mikewest** [comment]
>  hm, yes, IIUC, we ought to pick one of the above approaches and apply it to the other. It seems the "::" notation is not formally part of webidl at this time, see heycam/webidl#354, so perhaps we ought to go with the other approach (which is ugly...) ? Or maybe the "::" notation will soon return to webidl?

I think this was just me being a C++ programmer. Sorry about that! Let's discuss on the WebIDL bug.

**Comment by equalsJeffH** [comment]
bikeshed errors noted in https://github.com/w3c/webauthn/issues/414#issuecomment-298111117 are now gone (at least for me as of dda3e24

**Comment by equalsJeffH** [comment]
the inconsistency discussed in https://github.com/w3c/webauthn/issues/414#issuecomment-298119275 above was resolved in PR #464 i believe, so everything mentioned in this issue is now addressed AFAIK. Close it?

**Comment by kpaulh** [comment]
Yep! Thanks.
